src/components/ProjectCollaborators.tsx

```diff
@@ -1,23 +1,59 @@
-import { bpsToPercent } from '@/lib/utils';
-import { Database } from '@/types/supabase';
+import React, { useState } from 'react';
 
-type ProjectUserType = Database['mesa']['Tables']['project_users']['Row'];
+import UserMatrixCard from './ProjectMetaDataTable/UserMatrixCard';
+import { Button } from './ui/button';
+import { ProjectCollaboratorsProps, UserData } from '@/types/const';
 
-type ProjectType = Database['mesa']['Tables']['projects']['Row'] & {
-  project_users: ProjectUserType[];
-};
 
-export function ProjectCollaborators({project}: { project: ProjectType }) {
+export function ProjectCollaborators({ project }: ProjectCollaboratorsProps) {
+  const [currentPage, setCurrentPage] = useState(0);
+  const itemsPerPage = 3;
+  const totalPages = Math.ceil((project?.collaborators.length || 0) / itemsPerPage);
+
+  const handleNext = () => {
+    setCurrentPage(prevPage => Math.min(prevPage + 1, totalPages - 1));
+  };
+
+  const handlePrevious = () => {
+    setCurrentPage(prevPage => Math.max(prevPage - 1, 0));
+  };
+
+  const currentCollaborators = project?.collaborators.slice(
+    currentPage * itemsPerPage,
+    (currentPage + 1) * itemsPerPage,
+  ) as UserData[];
```

Comment:
let's remove pagination for now 🚮 

feels like it's adding unnecessary complexity when we should be focusing on `simple` solutions
```suggestion
```